capabilities/promtail/values/values.yaml

```diff
@@ -0,0 +1,164 @@
+            target_label: job
+          - action: replace
+            source_labels:
+              - __meta_kubernetes_namespace
+            target_label: namespace
+          - action: replace
+            source_labels:
+              - __meta_kubernetes_pod_name
+            target_label: pod
+          - action: replace
+            source_labels:
+              - __meta_kubernetes_pod_container_name
+            target_label: container
+          - replacement: /var/log/pods/*$1/*.log
+            separator: /
+            source_labels:
+              - __meta_kubernetes_pod_uid
+              - __meta_kubernetes_pod_container_name
+            target_label: __path__
+          - action: replace
+            source_labels:
+            - __meta_kubernetes_pod_node_name
+            target_label: node_name
+          - action: replace
+            source_labels:
+            - __meta_kubernetes_namespace
+            target_label: namespace
+          - action: replace
+            replacement: $1
+            separator: /
+            source_labels:
+            - namespace
+            - app
+            target_label: job
+          - action: replace
+            source_labels:
+            - __meta_kubernetes_pod_name
+            target_label: pod
+          - action: replace
+            source_labels:
+            - __meta_kubernetes_pod_container_name
+            target_label: container
+          - action: replace
+            replacement: /var/log/pods/*$1/*.log
+            separator: /
+            source_labels:
+            - __meta_kubernetes_pod_uid
+            - __meta_kubernetes_pod_container_name
+            target_label: __path__
+      - job_name: systemd-messages
+        static_configs:
+          - targets: [localhost]
+            labels:
+              job: varlogs
+              host: "${NODE_HOSTNAME}"
+              __path__: /var/log/*
+        relabel_configs:
+          - source_labels:
+            - __journal_systemd_unit
+            target_label: systemd_unit
+          - source_labels:
+            - __journal_hostname
+            target_label: nodename
+          - source_labels:
+            - __journal_syslog_identifier
+            target_label: syslog_identifier
+  
+
+containerSecurityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop:
+      - ALL
+  privileged: false
+  readOnlyRootFilesystem: true
+  runAsUser: 0
+  seLinuxOptions:
+    type: spc_t
+extraArgs:
+  - '-config.expand-env=true'
+
+extraEnv:
+  - name: NODE_HOSTNAME
+    valueFrom:
+      fieldRef:
+        fieldPath: spec.nodeName
+
+extraVolumes:
+  - hostPath:
+      path: /var/log
+    name: varlog
+  - hostPath:
+      path: /etc
+    name: machine-id
+
+extraVolumeMounts:
+  - mountPath: /var/log
+    name: varlog
+    readOnly: true
+  - mountPath: /etc/machine-id
+    name: machine-id
+    readOnly: true
+    subPath: machine-id
+
+resources:
+  limits:
+    cpu: 500m
```

Comment:
these limits seem a bit .... low? Are we sure this will be fine long-term?

Reply:
definitely not sure. That is what is set in the dubbd k3d deployment. I personally really like the idea of trying to find a better way to dynamically handle resource allocation with vpa or whatever else when it becomes an issue

Reply:
definitely not sure. That is what is set in the dubbd k3d deployment. I personally really like the idea of trying to find a better way to dynamically handle resource allocation with vpa or whatever else when it becomes an issue

Reply:
I think to be safe we should just set higher limits to avoid a needless pod termination doom-loop.

Reply:
i increased the limits for now, without data to back it though it feels pretty arbitrary.